packages/apps-config/src/api/spec/index.ts

```diff
@@ -69,7 +69,7 @@ import uart from './uart';
 import unique from './unique';
 import unitv from './unitv';
 import vln from './vln';
-import vlnrococo from './vln-rococo';
```

Comment:
Test failres due to this - Cannot find the `VLN-PC` map to `vlnrococo`

Reply:
propably happened during conflict resolution from my last upstream pull, guess I need to add these back in or how to resolve?

Reply:
added them back

Reply:
et voila :smile: